### ISSUE TITLE
Glossary: Outdated Unicode ISO Standard reference

### ISSUE THREAD

**Issue by ojeda** [rust-lang/fls#155]
ISO/IEC 10646:2017 is not the latest edition, 2020 is.

**Comment by PLeVasseur** [comment]
Possibly addressed by the linked, merged PR. @tshepang -- what do you think?

Also ping to @ojeda to see if that addresses the concern.

**Comment by ojeda** [comment]
No, this issue was about the Glossary, and from a quick look, it still refers to the older version. There are now amendments to the 2020 one from a quick look.

Having said that, given https://github.com/rust-lang/fls/issues/156, it may be simplest to avoid to refer to the ISO/IEC one to begin with, especially if Rust itself refers to the Unicode Standard. Plus the ISO/IEC may lag behind the other one anyway, as far as I understand.

**Comment by ojeda** [comment]
I sent a quick PR with a suggestion for both issues: https://github.com/rust-lang/fls/pull/617
